### ISSUE TITLE
Recurring exceptions when failing to load a targetBody that no longer exists

### ISSUE THREAD

**Issue by Sandriell** [jrossignol/ContractConfigurator#531]
Got this error after loading my save:

```
Exception occured while loading contract parameter 'HasAntenna' in contract 'RemoteTech.RT_OuterSat':
System.ArgumentException: 'Plock' is not a valid CelestialBody.
  at ContractConfigurator.ConfigNodeUtil.ParseCelestialBodyValue (System.String celestialName) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ConfigNodeUtil.ParseSingleValue[CelestialBody] (System.String key, System.String stringValue, Boolean allowExpression) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ConfigNodeUtil.ParseValue[CelestialBody] (.ConfigNode configNode, System.String key, Boolean allowExpression) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.ConfigNodeUtil.ParseValue[CelestialBody] (.ConfigNode configNode, System.String key, .CelestialBody defaultValue) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.RemoteTech.HasAntennaParameter.OnParameterLoad (.ConfigNode node) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.Parameters.ContractConfiguratorParameter.OnLoad (.ConfigNode node) [0x00000] in <filename unknown>:0 
```
And then got this one after I backed out to the main menu:
```
Exception occured while saving contract parameter 'ReachState' in contract 'Tourism.Tourism_GasGiantMoons':
System.NullReferenceException: Object reference not set to an instance of an object
  at ContractConfigurator.Parameters.ReachState.OnParameterSave (.ConfigNode node) [0x00000] in <filename unknown>:0 
  at ContractConfigurator.Parameters.ContractConfiguratorParameter.OnSave (.ConfigNode node) [0x00000] in <filename unknown>:0 
```

My mod list: http://pastebin.com/rBi0tkHc
KSP 1.1.3

**Comment by jrossignol** [comment]
Since Kopernicus isn't in your mod list, I assume you had it installed, and then removed it.  Unfortunately, Contract Configurator doesn't handle that too gracefully at the moment (see #425).  That being said, after you get through the mess of errors, those contracts should disappear and it should be smooth sailing.

**Comment by Sandriell** [comment]
No idea why it is not in the list, but I do have the latest version of Kopernicus (1.1.2)- had updated it right before loading.

**Comment by jrossignol** [comment]
Hmmm... I'd have to see the full KSP.log then.  Can you confirm that Plock
is there in the tracking station?

**Comment by Sandriell** [comment]
Yes, I can confirm it is there. Was the first thing I checked when I saw Plock mentioned in the error.

Here is the log file... captured right after loading my carear save and getting another contract Configurator error. https://gist.github.com/Sandriell/f9c6e26d7e653dfbd3bb13b2478dc2b3

**Comment by jrossignol** [comment]
Well, the Plock error is gone, but it keeps saving/loading the same broken contracts over again. I think I should be able to fix that.

**Comment by Sandriell** [comment]
Just wanted to give an update on this. Been upgrading mods as they have become available for 1.1.3 and this included a dev build for RemoteTech (build 554) which I assume is the one that would matter here. I also deleted my saves in my 1.1.3 copy of KSP and copied over my saves from 1.1.2 again.

I do not get any errors from Contract Configurator anymore.

**Comment by jrossignol** [comment]
For now I'm going to leave this one as is.  More often what happens is that it fails to load due to some issue that is quickly resolved, and the contracts then work.  Removing the contracts make it so that that issue can't be recovered from.  Given that seems to be the more common case, changing anything here will likely cause more problems than it solves.
